assets/js/accordeon.js

```diff
@@ -1,34 +1,34 @@
 /* ===== Zeste de Savoir ====================================================
    Accordeon for sidebar
    ---------------------------------
-   Author: Alex-D
+   Author: Alex-D, firm1
    ========================================================================== */
 
-(function($) {
-  'use strict'
+(function() {
+  function accordeon(element) {
+    const nextSearchElt = ['OL', 'UL']
 
-  function accordeon($elem) {
-    $('h4 + ul, h4 + ol', $elem).each(function() {
-      if (!$(this).hasClass('unfolded')) {
-        if ($('.current', $(this)).length === 0) { $(this).hide() }
-      }
-    })
+    Array.from(element.querySelectorAll('h4 + ul, h4 + ol'))
+      .filter(item => !item.classList.contains('unfolded'))
+      .filter(item => item.querySelectorAll('.current').length === 0)
+      .forEach(item => item.setAttribute('hidden', 'true'))
```

Comment:
L'attribut `hidden` ne devrait pas être utilisé ici. Il sert à indiquer qu'un élément est comme retiré de la page pour _tous les supports_, y compris les lecteurs d'écran par exemple, qui ne pourraient voir le contenu caché ainsi. Sémantiquement ce n'est pas du tout la même chose. D'ailleurs [ce cas d'usage est quasiment explicitement montré comme à éviter par le MDN](https://developer.mozilla.org/fr/docs/Web/API/HTMLElement/hidden) (ils parlent d'onglets, mais c'est le même esprit).

Il faudrait plutôt masquer avec du CSS en changeant une classe (par exemple `is-hidden`).